pkg/engine/engine.go

```diff
@@ -159,6 +164,23 @@ func (e *Engine) alterFuncMap(t *template.Template) template.FuncMap {
 		return val, nil
 	}
 
+	// Add the 'tpl' function here
+	funcMap["tpl"] = func(tpl string, vals chartutil.Values) (string, error) {
+		r := renderable{
+			tpl:  tpl,
+			vals: vals,
+		}
+
+		templates := map[string]renderable{}
+		templates["aaa_template"] = r
```

Comment:
I would not mind changing this to a randomly generated string just to discourage others from trying to access the template directly. But that's definitely not necessary for now.